src/answers-headless.ts

```diff
@@ -295,5 +295,15 @@ export default class AnswersHeadless {
     };
     this.stateManager.dispatchEvent('filters/toggleFacetOption', payload);
   }
+
+  toggleFilterOption(seletableFilter: SelectableFilter, filterCollectionId: string): void {
```

Comment:
nit: we may want to call this `setFilterOption`, because to me a toggle should already know what the current state is, and simply flip it. This seems to be more of a set since we are specifying whether or not the filter should be selected

Reply:
that make sense. updated